lldb/source/Symbol/SaveCoreOptions.cpp

```diff
@@ -46,8 +46,59 @@ SaveCoreOptions::GetOutputFile() const {
   return m_file;
 }
 
+void SaveCoreOptions::AddThread(lldb::tid_t tid) {
+  if (m_threads_to_save.count(tid) == 0)
+    m_threads_to_save.emplace(tid);
```

Comment:
No need to check the count, and no need to use `emplace` when we have a `std::set<lldb::tid_t>` and the type `T` is an integer:
```
m_threads_to_save.insert(tid);
```

Reply:
Ah, good catch. Originally I was using a vector but then moved to a set.